### ISSUE TITLE
How to connect to the https website? 

### ISSUE THREAD

**Issue by pengweichu** [whoshuu/cpr#194]
Hi all, I've compile the liburl as below:

`nmake /f Makefile.vc mode=static VC=14 WITH_SSL=static MACHINE=x64 RTLIBCFG=static`

And compile CPR as below:

a. Add that to a CMAKE_PREFIX_PATH environment variable and then it will find the system CURL.
b. Run the 2015 prompt command window, go to VS2015\VC, run "VCVARALL amd64" 
c.
```
git clone https://github.com/whoshuu/cpr
git submodule update --init
mkdir build
cmake -G "Visual Studio 14 2015 Win64" -DUSE_SYSTEM_CURL=ON -CMAKE_USE_OPENSSL=ON
```

Then I write a test project which linked liburl and CPR static libs, below code is works fine:

```
	auto r = cpr::Get(cpr::Url{ "http://www.xxx.com" });

	std::cout << r.url << std::endl; 
	std::cout << r.status_code << std::endl; // 200
	std::cout << r.header["content-type"] << std::endl; // application/json
	std::cout << r.text << std::endl;

```
But if change to https
`	auto r = cpr::Get(cpr::Url{ "https://www.xxx.com" }); `

Then the output just 0.

What's wrong with that ? How send GET to a https website ?

Thanks in advance.

**Comment by junwatu** [comment]
I have the same problem on windows 10. Try to get file from direct https URL and it just return empty response. i use vcpkg by the way.

**Comment by zjhmale** [comment]
exactly the same issue here on ubuntu@16.04

**Comment by zjhmale** [comment]
OK, one possible solution is using this https://github.com/whoshuu/cpr/commit/67d0b59e2eb3887afb4af3d34444f2ee32968c07#diff-632253cddea2ca6228f02cf09e1e9e80L49

need to change your code to:

```c++
	auto r = cpr::Get(cpr::Url{ "https://www.xxx.com" },
                          cpr::VerifySsl{false});

	std::cout << r.url << std::endl; 
	std::cout << r.status_code << std::endl; // 200
	std::cout << r.header["content-type"] << std::endl; // application/json
	std::cout << r.text << std::endl;
```

**Comment by COM8** [comment]
Should be fixed in [`cpr v1.5.0`](https://github.com/whoshuu/cpr/releases/tag/1.5.0).
Feel free to open a new issue, in case this is still a problem.
